src/luxonis_ml/data/dataset.py

```diff
@@ -631,13 +661,26 @@ def _add_filter(self, additions):
                     for change in changes:
                         field, value = list(change.items())[0]
                         field_change = True
-                        tid = self._make_transaction(
-                            LDFTransactionType.UPDATE,
-                            sample_id=latest_sample._id,
-                            field=field,
-                            value=value,
-                            component=component_name,
-                        )
+                        if field == "split":
+                            group = data_utils.get_group_from_sample(
+                                self, latest_sample
+                            )
+                            for component_name in group:
```

Comment:
Does this make the updates slower?

Reply:
Yes it will. I think probably not by much though and only in the case where split is updated.